File: app/src/test/java/com/gmail/fattazzo/formula1world/MainAppTest.java
package com.gmail.fattazzo.formula1world;

import org.junit.Assert;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.RobolectricTestRunner;
import org.robolectric.RuntimeEnvironment;

@RunWith(RobolectricTestRunner.class)
public class MainAppTest {

    @Test
    public void runtimeApplicationShouldBeTestApp() throws Exception {
        // !!! Add MODULE_DIR in Run --> Edit Configurations --> Working directory
        String actualName = RuntimeEnvironment.application.getClass().getName();
        String expectedName = TestApp.class.getName();

        Assert.assertTrue((actualName).equals(expectedName));
    }
}

File: app/src/test/java/com/gmail/fattazzo/formula1world/MainAppTest.kt
package com.gmail.fattazzo.formula1world

import org.junit.Assert
import org.junit.Test
import org.junit.runner.RunWith
import org.robolectric.RobolectricTestRunner
import org.robolectric.RuntimeEnvironment

@RunWith(RobolectricTestRunner::class)
class MainAppTest {

    @Test
    @Throws(Exception::class)
    fun runtimeApplicationShouldBeTestApp() {
        // !!! Add MODULE_DIR in Run --> Edit Configurations --> Working directory
        val actualName = RuntimeEnvironment.application.javaClass.name
        val expectedName = TestApp::class.java.name

        Assert.assertTrue(actualName == expectedName)
    }
}